internal/handler/impl/application_test.go

```diff
@@ -6,38 +6,59 @@ import (
 	"github.com/podengo-project/idmsvc-backend/internal/config"
 	"github.com/podengo-project/idmsvc-backend/internal/metrics"
 	"github.com/podengo-project/idmsvc-backend/internal/test"
-	client_inventory "github.com/podengo-project/idmsvc-backend/internal/test/mock/interface/client/inventory"
+	"github.com/podengo-project/idmsvc-backend/internal/test/mock/interface/client/pendo"
+	"github.com/podengo-project/idmsvc-backend/internal/test/mock/interface/client/rbac"
 	// client_rbac "github.com/podengo-project/idmsvc-backend/internal/test/mock/interface/client/rbac"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"gorm.io/gorm"
 )
 
 func TestNewHandler(t *testing.T) {
+	assert.PanicsWithValue(t, "'cfg' is nil", func() {
+		NewHandler(nil, nil, nil, nil, nil)
+	})
+
+	cfg := test.GetTestConfig()
+	assert.PanicsWithValue(t, "'db' is nil", func() {
+		NewHandler(&config.Config{}, nil, nil, nil, nil)
+	})
+
 	sqlMock, gormDB, err := test.NewSqlMock(&gorm.Session{SkipHooks: true})
-	inventoryMock := client_inventory.NewHostInventory(t)
 	require.NoError(t, err)
 	require.NotNil(t, sqlMock)
 	require.NotNil(t, gormDB)
-	assert.Panics(t, func() {
-		NewHandler(nil, nil, nil, nil, nil, nil)
+	assert.PanicsWithValue(t, "'m' is nil", func() {
+		NewHandler(cfg, gormDB, nil, nil, nil)
 	})
-	assert.Panics(t, func() {
-		NewHandler(&config.Config{}, nil, nil, nil, nil, nil)
+
+	m := &metrics.Metrics{}
+	assert.PanicsWithValue(t, "'rbac' is nil", func() {
+		NewHandler(cfg, gormDB, m, nil, nil)
 	})
-	cfg := test.GetTestConfig()
+
+	rbacClient := rbac.NewRbac(t)
+	assert.PanicsWithValue(t, "'pendo' is nil", func() {
+		NewHandler(cfg, gormDB, &metrics.Metrics{}, rbacClient, nil)
```

Comment:
nit: shouldn't this use `m` instead of `&metrics.Metrics{}` - to have consistent style. Same comment applies also for next call.